Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+                # Write centerbody profile coordinates, using a tab delimiter
+                for row in xy_centerbody:
+                    file.write('    '.join(map(str, row)) + '\n')
+                
+                # Write separator line, using a tab delimiter
+                file.write('    '.join(map(str, [999., 999.])) + '\n')
+
+                # Write duct profile coordinates, using a tab delimiter
+                for row in xy_duct:
+                    file.write('    '.join(map(str, row)) + '\n')
+
+            return None
```

Ruff (0.8.2)

Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)